test/unit/test.xpi.js

```diff
@@ -45,13 +46,29 @@ describe("lib/xpi", function() {
     })
     .catch(done);
   });
+  it("Check file name with id", function(done) {
```

Comment:
You don't need to declare `done` here. Just return the promise (as you are) and the test will complete when the promise completes or it will fail when it's rejected.